projects/components/src/multi-select/multi-select.component.scss

```diff
@@ -80,11 +80,16 @@
     height: 34px;
     padding: 4px 16px;
     cursor: pointer;
-    font-size: 14px;
+    font-size: 15px;
     align-items: center;
 
     .checkbox {
       margin: 0px;
+      color: $gray-3;
+
+      ::ng-deep .mat-ripple {
+        display: none;
```

Comment:
Do we not want the ripple here? I don't have strong feelings one way or another, but it seems like the ripple should be consistent across all our checkboxes - inside a multiselect or otherwise.

Reply:
I'm ignoring the ripple there because I don't want to have the hover effect because the complete line has its own hover.